### PR TITLE
WFLY-15642 Kitchen Sink quickstart uses outdated EAP logo

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-15642

### CODE REVIEW

**Comment by jamezp** [comment]
I'm not sure this is quite right for a community project. Also the logo is massive so it would need something like `width="300"` on the `img` element.

**Comment by RanabirChakraborty** [comment]
An issue was created for `jboss-eap-quickstart`, so I thought to change here as well. I can change the width but If you think for the community project it's not required, then I can close the PR.

**Comment by jamezp** [comment]
I still think we should be using the community logo here.

**Comment by emmartins** [comment]
It should be a logo good for both Wildfly and EAP

**Comment by RanabirChakraborty** [comment]
@emmartins if possible can you please suggest to me any logo?

**Comment by emmartins** [comment]
@RanabirChakraborty sorry but missed notification of your question, perhaps the JBoss by Red Hat logo?

**Comment by RanabirChakraborty** [comment]
@emmartins will this be good to use? - [here](https://upload.wikimedia.org/wikipedia/commons/9/95/JBoss_logo.svg)

**Comment by emmartins** [comment]
@RanabirChakraborty IMO yes

File: hibernate/src/main/webapp/WEB-INF/templates/default.xhtml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 <h:body>
    <div id="container">
       <div class="dualbrand">
-         <img src="resources/gfx/rhjb_eap_logo.png" />
+         <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60" />
```
Comment (emmartins) [inline hibernate/src/main/webapp/WEB-INF/templates/default.xhtml:30]: The height should be 175, to keep the logo's original aspect ratio.

File: kitchensink-angularjs/src/main/webapp/index.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
 <body>
     <aside id="container">
         <div class="dualbrand">
-            <img src="gfx/rhjb_eap_logo.png" />
+            <img src="gfx/rhjb_eap_logo.png" width="300" height="60" />
```
Comment (emmartins) [inline kitchensink-angularjs/src/main/webapp/index.html:47]: The height should be 175, to keep the logo's original aspect ratio.

File: kitchensink-ear/web/src/main/webapp/WEB-INF/templates/default.xhtml [hunk 3]
```diff
@@ -27,7 +27,7 @@
 <h:body>
    <div id="container">
       <div class="dualbrand">
-         <img src="resources/gfx/rhjb_eap_logo.png" />
+         <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60"/>
```
Comment (emmartins) [inline kitchensink-ear/web/src/main/webapp/WEB-INF/templates/default.xhtml:30]: The height should be 175, to keep the logo's original aspect ratio.

File: kitchensink-jsp/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -27,7 +27,7 @@
 <body>
     <div id="container">
         <div align="right" class="dualbrand">
-            <img src="resources/gfx/rhjb_eap_logo.png" />
+            <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60" />
```
Comment (emmartins) [inline kitchensink-jsp/src/main/webapp/index.jsp:30]: The height should be 175, to keep the logo's original aspect ratio.

File: kitchensink-ml/src/main/webapp/WEB-INF/templates/default.xhtml [hunk 5]
```diff
@@ -28,7 +28,7 @@
 <h:body>
     <div id="container">
         <div class="dualbrand">
-            <img src="resources/gfx/rhjb_eap_logo.png" />
+            <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60"/>
```
Comment (emmartins) [inline kitchensink-ml/src/main/webapp/WEB-INF/templates/default.xhtml:31]: The height should be 175, to keep the logo's original aspect ratio.

File: kitchensink/src/main/webapp/WEB-INF/templates/default.xhtml [hunk 6]
```diff
@@ -27,7 +27,7 @@
 <h:body>
     <div id="container">
         <div class="dualbrand">
-            <img src="resources/gfx/rhjb_eap_logo.png" />
+            <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60" />
```
Comment (emmartins) [inline kitchensink/src/main/webapp/WEB-INF/templates/default.xhtml:30]: The height should be 175, to keep the logo's original aspect ratio.

File: payment-cdi-event/src/main/webapp/WEB-INF/templates/template.xhtml [hunk 7]
```diff
@@ -31,7 +31,7 @@
    <div class="container-fluid">
             
       <div class="dualbrand">
-         <img src="resources/gfx/rhjb_eap_logo.png" />
+         <img src="resources/gfx/rhjb_eap_logo.png" width="300" height="60"/>
```
Comment (emmartins) [inline payment-cdi-event/src/main/webapp/WEB-INF/templates/template.xhtml:34]: The height should be 175, to keep the logo's original aspect ratio.

File: spring-kitchensink-basic/src/main/webapp/WEB-INF/views/index.jsp [hunk 8]
```diff
@@ -28,7 +28,7 @@
 <body>
 <div id="container">
     <div class="dualbrand">
-        <img src="<c:url value="/static/resources/gfx/rhjb_eap_logo.png"/>"/>
+        <img src="<c:url value="/static/resources/gfx/rhjb_eap_logo.png"/>" width="300" height="60"/>
```
Comment (emmartins) [inline spring-kitchensink-basic/src/main/webapp/WEB-INF/views/index.jsp:31]: The height should be 175, to keep the logo's original aspect ratio.

File: spring-kitchensink-springmvctest/src/main/webapp/WEB-INF/views/index.jsp [hunk 9]
```diff
@@ -28,7 +28,7 @@
 <body>
 <div id="container">
     <div class="dualbrand">
-        <img src="<c:url value="/static/resources/gfx/rhjb_eap_logo.png"/>"/>
+        <img src="<c:url value="/static/resources/gfx/rhjb_eap_logo.png"/>" width="300" height="60"/>
```
Comment (emmartins) [inline spring-kitchensink-springmvctest/src/main/webapp/WEB-INF/views/index.jsp:31]: The height should be 175, to keep the logo's original aspect ratio.

**Review by emmartins — CHANGES_REQUESTED** [review]

**Comment by RanabirChakraborty** [comment]
Thanks, @emmartins Did the changes :)

**Comment by RanabirChakraborty** [comment]
@emmartins Can you please take a look.

**Pull request merged**